### DISCUSSION TITLE
Distinct Zero

### DISCUSSION THREAD

**Discussion by unrau** [IBM/plex#533] [Ideas]
IBM Plex has quickly become my favourite font, and the biggest benefit for me is how it distinguishes upper-case Is from lower-case ls, even viewed in isolation, and with serifed upper-case Is even in the sans-serif typeface. I think it would be a further benefit if a zero more distinct from o, O, and Ø were included, and not just as OpenType stylistic options.

For a font without a slashed zero for U+0030, copy/pasting a slashed zero produces a combining sequence, which often displays incorrectly. The image attached shows the slashed zero pasted into Apple's TextEdit, and into Microsoft's OneNote, both using the current OpenType v6.3.0 of IBM Plex Sans Regular.

A similar question about access to slashed and dotted zero glyphs was asked here:
https://github.com/IBM/plex/discussions/358
But most applications still can't access OpenType stylistic features.

I'd like to suggest making Plex's default U+0030 a slashed zero.

Is there a reason perhaps that the current form of 0 was chosen? Does anyone else think it should be slashed by default?

Thank you for hearing my humble suggestion, and thank you for making this stunning font!

![SCR-20230911-irp](https://github.com/IBM/plex/assets/5354791/c3953e2a-c515-4ad7-87b0-905850ca2f59)
![SCR-20230911-ifz](https://github.com/IBM/plex/assets/5354791/c65f8b3d-51c6-4c3a-9df4-051b4f29f29c)

**Comment by BoldMonday** [comment]
I did not know that `zero` with `combining slash` is used to create a slashed zero. What other typefaces do support this?

Changing the default `zero` to a slashed version is out of the question. It has been chosen very deliberately and since the proportions between `zero` and uppercase `O` are different, there is little cause for confusion. And even if we make slashed `zero` default, your argument can just as much be applied back to uppercase `Ø`.

I do find the suggestion useful to cater for the combination of `zero` with `combining slash` though.
But be aware that this also requires OpenType positioning rules. It will not work in environments that do not support OpenType features.

**Reply by unrau** [reply]
Hi, thank you for responding,

I only found out about the zero + slash combining sequence because of the issue with OneNote displaying it with Plex. I don't know what other typefaces might support it. I got it by copying the slashed zero character from [this Wikipedia page](https://en.wikipedia.org/wiki/Slashed_zero), and then pasting it into a document using the Plex font.

I can certainly appreciate that the current zero has been deliberately chosen, and you make a good point about a slashed zero being very similar to `Ø`. Utmost respect to yourself and everyone who has put work into this beautiful and functional font <3

Thank you.
